ams/server/src/main/java/com/netease/arctic/server/dashboard/controller/OptimizerController.java

```diff
@@ -114,7 +114,7 @@ public void getOptimizers(Context ctx) {
         optimizerList.stream()
             .map(
                 e -> {
-                  ObjectNode jsonObject = (ObjectNode) JacksonUtil.fromObjects(e);
+                  ObjectNode jsonObject = new ObjectMapper().valueToTree(e);
```

Comment:
We should import the shaded classes as much as possible, so it's best to construct a POJO for the optimizer instance.

Reply:
> We should import the shaded classes as much as possible, so it's best to construct a POJO for the optimizer instance.

Make sense, I also suggest append the optimier token and display it on webpage, it can help us identifier optimizers

Reply:
LGTM.

Reply:
Thanks for your comments. Plz have a look again.